appveyor.yml

```diff
@@ -31,4 +34,5 @@ install:
 
 artifacts:
  - path: bin
-   name: rpcs3-$(configuration)-$(COMMIT_SHA)
+   name: 'rpcs3-v0.0.1-$(COMMIT_SHA)_$(Date)_win64'
```

Comment:
I suggest to swap SHA and Date (so date is before sha), so if there are several builds on the same day, it'd be easier to sort them by name.